packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div
+    data-cy="debug"
+    class="flex flex-col pb-24px gap-16px"
+  >
+    <div
+      :data-cy="`debug-header-${debug.id}`"
```

Comment:
The variable number on the end of this `data-cy` was used on the Runs page because there were multiple Runs being displayed.  That is not needed here.